lib/jvertica.rb

```diff
@@ -157,262 +168,10 @@ def copy_stream query, io = nil, &blk
-      end
-    end
-
-    def each(&blk)
-      @values.each do |v|
-        yield v
-      end
-    end
-
-    def inspect
-      strs = []
-      @labels.each do |col|
-        strs << "#{col}: #{self[col] || '(null)'}"
-      end
-      '[' + strs.join(', ') + ']'
-    end
-
-    def to_a
-      @values
-    end
-
-    def join sep = $OUTPUT_FIELD_SEPARATOR
-      to_a.join sep
-    end
-
-    def eql? other
-      self.hash == other.hash
-    end
-
-    def hash
-      @labels.zip(@values).sort.hash
-    end
-
-    def to_h
-      Hash[@labels.zip @values]
-    end
-
-    alias :== :eql?
-
-    def initialize col_labels, col_labels_d, values, rownum
-      @labels = col_labels
-      @labels_d = col_labels_d
-      @values = values
-      @rownum = rownum
-    end
-
-    def method_missing symb, *args
-      if vidx = @labels_d.index(symb.to_s.downcase)
-        @values[vidx]
-      elsif @values.respond_to? symb
-        @values.send symb, *args
-      else
-        raise NoMethodError.new("undefined method or attribute `#{symb}'")
-      end
-    end
-
-    [:id, :tap, :gem, :display, :class, :method, :methods, :trust].select do |s|
-      method_defined? s
-    end.each do |m|
-      undef_method m
-    end
-  end
-
-  class DriverManager < java.sql.DriverManager
-  end
-
-  class Properties < java.util.Properties
-  end
-
-  class DataSource < com.vertica.jdbc.DataSource
-  end
-
-  module Constant
-    CONNECTION_PROPERTY_DOCUMENT_URL =
-      'http://my.vertica.com/docs/7.1.x/HTML/index.htm#Authoring/ProgrammersGuide/ClientJDBC/JDBCConnectionProperties.htm'
-
-    RUBY_SQL_TYPE_MAP = {
-      Fixnum => java.sql.Types::INTEGER,
-      Bignum => java.sql.Types::BIGINT,
-      String => java.sql.Types::VARCHAR,
-      Float => java.sql.Types::DOUBLE,
-      Time => java.sql.Types::TIMESTAMP
-    }
-    GETTER_MAP = {
-      java.sql.Types::TINYINT => :getInt,
-      java.sql.Types::SMALLINT => :getInt,
-      java.sql.Types::INTEGER => :getInt,
-      java.sql.Types::BIGINT => :getLong,
-      java.sql.Types::CHAR => :getString,
-      java.sql.Types::VARCHAR => :getString,
-      java.sql.Types::LONGVARCHAR => :getString,
-      (java.sql.Types::NCHAR rescue nil) => :getString,
-      (java.sql.Types::NVARCHAR rescue nil) => :getString,
-      (java.sql.Types::LONGNVARCHAR rescue nil) => :getString,
-      java.sql.Types::BINARY => :getBinaryStream,
-      java.sql.Types::VARBINARY => :getBinaryStream,
-      java.sql.Types::LONGVARBINARY => :getBinaryStream,
-      java.sql.Types::REAL => :getDouble,
-      java.sql.Types::FLOAT => :getFloat,
-      java.sql.Types::DOUBLE => :getDouble,
-      java.sql.Types::DATE => :getDate,
-      java.sql.Types::TIME => :getTime,
-      java.sql.Types::TIMESTAMP => :getTimestamp,
-      java.sql.Types::BLOB => :getBlob,
-      java.sql.Types::CLOB => :getString,
-      (java.sql.Types::NCLOB rescue nil) => :getString,
-      java.sql.Types::BOOLEAN => :getBoolean
-    }
-  end
-end
-
-class Hash
```

Comment:
Should not open class in a gem like this (We know ActiveSupport does open class, but other gems should not do it normally)